documentation/bolt.ditamap

```diff
@@ -20,7 +20,18 @@
         <topicref href="privilege_escalation.md" format="markdown"/>
         <topicref href="vscode_and_bolt.md" format="markdown"/>
     </topichead>
-    <topicref href="running_bolt_commands.md" format="markdown" linking="targetonly"/>
+    <topicref href="running_bolt.md" format="markdown" navtitle="Running Bolt" locktitle="yes">
```

Comment:
Honestly I'm all for making it a topichead. I think I just copied the plans section maybe, since it seemed similar, but I definitely expect that those "links" just expand the subpages and aren't pages themselves.

Reply:
Okay cool - the added benefit of using a topichead is that we don't need the short descriptions :D